python-requirements.txt

```diff
@@ -7,3 +7,6 @@ git+https://github.com/lowRISC/edalize.git@ot
 
 # Development version with OT-specific changes
 git+https://github.com/lowRISC/fusesoc.git@ot
+
+pyyaml
+mako
```

Comment:
Is this for `primgen`? Does that now get run as part of an Ibex build?

Reply:
Yes. (pyyaml is also used for ibex_config.py but hasn't been included in the requirements file so far and was installed on CI runners by "luck" [not real luck, it's a dependency of fusesoc and edalize])